modules/sqlite3/3.46.0/patches/add_build_file.patch

```diff
@@ -0,0 +1,32 @@
+--- /dev/null
++++ BUILD.bazel
+@@ -0,0 +1,29 @@
++cc_binary(
++    name = "shell",
++    srcs = ["shell.c"],
++    # Disable Bazel's default behavior of linking to `libstdc++` / `libc++`.
++    features = ["-default_link_libs"],
++    deps = [":sqlite3"],
++)
++
++cc_library(
++    name = "sqlite3",
++    srcs = ["sqlite3.c"],
++    hdrs = ["sqlite3.h"],
++    # Some Unix platforms have pthread and/or dlopen in separate libraries.
++    linkopts = select({
++        "@platforms//os:freebsd": ["-lpthread"],
++        "@platforms//os:linux": [
++            "-lpthread",
++            "-ldl",
++        ],
++        "@platforms//os:netbsd": ["-lpthread"],
++        "@platforms//os:openbsd": ["-lpthread"],
++        "//conditions:default": [],
++    }),
++)
```

Comment:
I imagine you'll want `visibility = ["//visibility:public"]` on these targets as well